frontend/src/Components/Filter/Builder/ReleaseStatusFilterBuilderRowValue.js

```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import translate from 'Utilities/String/translate';
+import FilterBuilderRowValue from './FilterBuilderRowValue';
+
+const protocols = [
+  { id: 'tba', name: 'TBA' },
+  { id: 'announced', name: translate('Announced') },
+  { id: 'inCinemas', name: translate('InCinemas') },
+  { id: 'released', name: translate('Released') },
+  { id: 'deleted', name: translate('Deleted') }
```

Comment:
Is deleted necessary? Not sure the use case of it as a Release Status. If the movie is announced and then cancelled it just wouldn't be listed on TMDb and if it did get listed and people added it, once they delete it Radarr will just show it as missing and not deleted.

Reply:
I didn't actually change the file ReleaseStatusFilterBuilderRowValue, other than rename from Status to ReleaseStatus and add in the translations since I added the translations to MinimumAvailabilityFilterBuilderRowValue which is very similar to ReleaseStatusFilterBuilderRowValue. If you notice in MinimumAvailabilityFilter I did not include the Deleted - because I know for MinimumAvailability Deleted is not needed.

For ReleaseStatus the Deleted was there prior to my changes- I also noticed it and I'm not sure if it is needed - but don't really want to remove it as part of this PR - since it is unrelated - its just there already.

Reply:
I opened an issue to track the cleaning up of these deprecated or unneeded MovieStatus Types:
https://github.com/Radarr/Radarr/issues/5002

Reply:
Deleted is used for movies that are removed from TMDB but still exist in Radarr instance, it means deleted from metadata, not from your instance. It shouldn’t be removed

Reply:
OK - thats fine with me - i just added a comment here: https://github.com/Radarr/Radarr/issues/5002#issuecomment-691791275 - I still will look into removing TBA and PreDb etc.